From 1c41767c65ae82b3d33442a0c306132e62ab78bc Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Tue, 12 May 2026 10:12:17 -0500
Subject: [PATCH 127/268] win32u: Silence
 DISPLAYCONFIG_DEVICE_INFO_GET_ADVANCED_COLOR_INFO fixme.

---
 dlls/win32u/sysparams.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dlls/win32u/sysparams.c b/dlls/win32u/sysparams.c
index 49a376d547d..2515b460161 100644
--- a/dlls/win32u/sysparams.c
+++ b/dlls/win32u/sysparams.c
@@ -8072,10 +8072,11 @@ NTSTATUS WINAPI NtUserDisplayConfigGetDeviceInfo( DISPLAYCONFIG_DEVICE_INFO_HEAD
     }
     case DISPLAYCONFIG_DEVICE_INFO_GET_ADVANCED_COLOR_INFO:
     {
+        static int once;
         DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO *color_info = (DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO *)packet;
         struct monitor *monitor;
 
-        FIXME( "DISPLAYCONFIG_DEVICE_INFO_GET_ADVANCED_COLOR_INFO semi-stub.\n" );
+        if (!once++) FIXME( "DISPLAYCONFIG_DEVICE_INFO_GET_ADVANCED_COLOR_INFO semi-stub.\n" );
 
         if (packet->size < sizeof(*color_info))
             return STATUS_INVALID_PARAMETER;
-- 
2.55.0

